test/file_io.jl

```diff
@@ -152,4 +152,27 @@ end
             @test result[k] == v
         end
     end
+    @testset "keys are not Symbols" begin
+        @test_throws(
+            getlogger(JLSO),
+            MethodError,
+            JLSO.save("breakfast.jlso", "food" => "☕️🥓🍳", "time" => Time(9, 0)),
+        )
+    end
+    @testset "README example" begin
+        mktempdir() do path
+            JLSO.save(
+                "$path/breakfast.jlso",
```

Comment:
```suggestion
                joinpath(path, "breakfast.jlso"),
```

Reply:
thanks :) if using FilePaths, would be even nicer: `path / "breakfast.jlso"`